test/Java9andUp/src/org/openj9/test/java9AttachAPI/SelfAttacher.java

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2015, 2017 IBM Corp. and others
```

Comment:
As this is new code the start date should be 2017

Reply:
Fixed.